### PR TITLE
add iofs test cases bat file

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: scripts/bat/ofs/ofs_bat.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+#
+# IOFS EA Test cases
+
```
Comment (tswhison) [inline scripts/bat/ofs/ofs_bat.sh:4]: You could use
```bash
set -e
```
to exit on the first error

**Comment by buie1** [comment]
Let's update this branch and any idea why Travis is failing?

File: scripts/bat/ofs/ofs_bat.sh [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+ make -j
+ OPAE_EXPLICIT_INITIALIZE=1 LD_LIBRARY_PATH=$PWD/lib make test
+ echo "--END HW UNIT TESTS--"
+
+ echo "--START CREATE RPM--"
+ rm -rf *
+ cmake ..   -DCPACK_GENERATOR=RPM -DOPAE_BUILD_LEGACY=ON -DOPAE_BUILD_SIM=ON
+ make -j
+ make package_rpm
+ yum -y remove opae*
```
Comment (tswhison) [inline scripts/bat/ofs/ofs_bat.sh:27]: This locks it into a distribution.  Do we need to be agnostic?

File: scripts/bat/ofs/ofs_bat.sh [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+
+found=$(ls /dev/dfl-* | wc -l)
+echo $found
+if [[ $found -lt 1 ]]; then
+    echo "Failed to locate drivers."
+    exit 1
+fi
+
+found=$(lspci | grep 0b2b)
```
Comment (tswhison) [inline scripts/bat/ofs/ofs_bat.sh:42]: This locks it into a platform.  Do we need to be agnostic?

File: scripts/bat/ofs/ofs_bat.sh [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+else
+  bus_num=${found:0:2}
+  if [[ $((16#$bus_num)) -lt 0 ]]; then
+      echo "Invalid bus number: "$bus_num
+      exit 1
+  fi
+fi
+
+gbs_file=nlb_mode_0_unsigned.gbs
+iofs_file=ofs_fim_page1_unsigned.bin
```
Comment (tswhison) [inline scripts/bat/ofs/ofs_bat.sh:59]: Is it safe to assume that these files will exist in the current working directory?

File: scripts/bat/ofs/ofs_bat.sh [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+"fpgainfo temp -B 0x$bus_num"
+"fpgainfo bmc -B 0x$bus_num"
+"fpgainfo power -B 0x$bus_num"
+"fpgainfo errors all -B 0x$bus_num"
+"fpgainfo security -B 0x$bus_num"
+"hello_events -B 0x$bus_num"
+"packager gbs-info --gbs $gbs_file"
+"fpgametrics -B 0x$bus_num"
+"bitstreaminfo -v  $iofs_file"
+"(echo Y && echo Y) |sudo -u lab PACSign SR -t update -H openssl_manager  -i ofs_fim_page1_unsigned.bin  -o test_fim.bin -r darby_dev_fim_root_public_256.pem  -k darby_dev_fim_csk0_public_256.pem"
```
Comment (tswhison) [inline scripts/bat/ofs/ofs_bat.sh:81]: Assumption on bin files in the current working directory.

File: scripts/bat/ofs/ofs_bat.sh [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+"echo yes | fpgasupdate   darby_dev_fim_root_hash.bin"
```
Comment (tswhison) [inline scripts/bat/ofs/ofs_bat.sh:82]: Assumption on bin file in the current working directory.

File: scripts/bat/ofs/ofs_bat.sh [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+"fpgainfo security -B 0x$bus_num"
+"echo yes | fpgasupdate  darby_dev_bmc_root_hash.bin"
```
Comment (tswhison) [inline scripts/bat/ofs/ofs_bat.sh:84]: Assumption on bin file in the current working directory.

File: scripts/bat/ofs/ofs_bat.sh [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+"fpgainfo security -B 0x$bus_num"
+"hello_fpga -B 0x$bus_num"
+"fpgaconf -B 0x$bus_num -V gbs_file"
```
Comment (tswhison) [inline scripts/bat/ofs/ofs_bat.sh:87]: I think you mean to dereference gbs_file here, so it needs the $, as in
```bash
$gbs_file
```

**Pull request closed without merging**